daml-lf/transaction/src/main/scala/com/digitalasset/daml/lf/transaction/Node.scala

```diff
@@ -227,6 +229,7 @@ object Node {
     *     actual validation stage.
     */
   final case class NodeExercises[+Nid, +Cid, +Val](
+      observers: Set[Party], //additional choice observers. -- NICK: re-order after signatories
```

Comment:
When computing the witnesses of an exercise node, you'll need to consider the template observers and the choice observers. Thus, I would prefer to call this field `choiceObservers` (or `informees` if we go with that) to avoid any confusion.

@remyhaemmerle-da Do we need to make this field optional for the sake of backward compatibility?

Reply:
I would not speak of "additional choice observers". It should either be "choice observers" or "additional informees".